### PR TITLE
[13.0][ADD] web_clickjack_protection

### PR DESCRIPTION
**Pull request by KKamaa**
This module tries to mitigate issues arising from click-jacking attacks. To test this well,  go to [niutech](https://github.com/niutech/x-frame-bypass) and read on [OWASP clickjacking cheat sheet](https://cheatsheetseries.owasp.org/cheatsheets/Clickjacking_Defense_Cheat_Sheet.html) v 13.0

### CODE REVIEW

File: web_clickjack_protection/readme/USAGE.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Just install
```
Comment (legalsylvain) [inline web_clickjack_protection/readme/USAGE.rst]: I think this file is not necessary

Comment (KKamaa) [reply web_clickjack_protection/readme/USAGE.rst]: Added it as a simple explainer for those that might wonder if there is any prerequisites needed e.g widget property.

Comment (KKamaa) [reply web_clickjack_protection/readme/USAGE.rst]: Fixed as per OCA standard.

**Review by bofiltd — APPROVED** [review]

**Review by bosd — APPROVED** [review]
Quick code review LGTM :+1: 
Would love to see this one on a more recent version 14 or above.

**Comment by bosd** [comment]
@KKamaa Can you please change the title of this PR to include the version number e.g. [13.0]

**Comment by KKamaa** [comment]
> Contributor

I changed it to show Odoo version

**Pull request closed without merging**